compiler/luci/service/src/Nodes/CirclePad.cpp

```diff
@@ -15,6 +15,9 @@
  */
 
 #include "CircleCloneNode.h"
+#include "CircleShapeInferenceHelper.h"
+
+#include "luci/Service/CircleShapeInference.h"
```

Comment:
@icodo98 , please check https://github.com/Samsung/ONE/pull/13784/files#r1732344945 and
can you please update this include orders too?

Reply:
> @icodo98 , please check https://github.com/Samsung/ONE/pull/13784/files#r1732344945 and can you please update this include orders too?

Yes, I'll update include orders.